### ISSUE TITLE
System Interface advancement & Playlist Improvement (not FreeTube Playlist Panel)

### ISSUE THREAD

**Issue by bachig26** [FreeTubeApp/FreeTube#1963]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
1. The **color contrast is way too high** and it **cannot be changed**, which can be verified between the screenshots provided. **The present visual looks whitewashed or polished. This affects the eyes during night times, and need a proper night mode.**

2. The **outline of the layout throughout the system is a mess and feels out of order, the enclosure of the playlist contents and recommendations overlay each other and the layout is not properly defined**. This also can be verified from the screenshots if you can look at the playlist videos box and recommendations. Also, **there is no deep between the layout of each section to make a conclusive difference**.

3. **With the Playlist, the video player doesn't have a back and forward button to skip to and fro between playlist videos**. This function needs to be necessary for the playlist contents. As, **this is a big problem during Fullscreen view, as it needs to be window mode to use the existing skip function.**

4. Also, **a translation function is needed similar to YouTube, to view the texts in system language or user-defined language.** This difference also can understand between the provided screenshots.

To understand the exact difference, I have attached the same Playlist from YouTube and FreeTube below.

**Screenshots** 
The screenshots from the [same playlist link](https://www.youtube.com/watch?v=3gC5cZBumpg&list=PLATwx1z00HseAoW7iJL141nbbi6KmxHEq&index=4).
From YouTube
![vivaldi_OJzGeLMuPN](https://user-images.githubusercontent.com/96393348/146862717-d147b2c8-4aeb-45ad-abd4-7cedfc382237.png)

From FreeTube
![FreeTube_g5QlbCVdAy](https://user-images.githubusercontent.com/96393348/146862731-c75c1205-ad60-486e-a2c0-e6716eec24d2.png)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
1. We have dark mode and black mode. I think that mode will fit ur needs.

2. Duplicate of #856 23. on the list

3. Duplicate of #1220

4. Duplicate of #1801

Search for duplicate issues before opening ur issues! If there already is an issue related to the subject urs will get closed because it is cluttering the backlog.

Please do not open an issue and put multiple issues into it! This makes it very hard for us to analyse and will also get closed for not following the rules.
